src/defaults/templates/articleDocumentTemplate.ts

```diff
@@ -1,17 +1,61 @@
 import { Block, Document } from '@ttab/elephant-api/newsdoc'
+import { ungroup } from '../../../src-srv/utils/transformations/groupedNewsDoc'
```

Comment:
We really should refactor all these functions to shared, but maybe not now...

Reply:
Yeah really